webapp/i18n/ja.json

```diff
@@ -718,7 +718,6 @@
   "channel_notifications.unreadInfo": "チャンネル名は未読のメッセージがある場合、サイドバーで太字で表示されます。「あなたについての投稿のみ」を選択することで、あなたについての投稿がある場合のみ太字で表示されます。",
   "channel_select.placeholder": "--- チャンネルを選択してください ---",
   "channel_switch_modal.help": "↑↓ to browse, TAB to select, ↵ to confirm, ESC to dismiss",
```

Comment:
@hmhealey that is working as expected, since the translation server adds the english string in case there is no translation yet for that string, once it get translated then it is replaced with the correct one.